src/IdeaStatiCa.RcsClient/Factory/RcsClientFactory.cs

```diff
@@ -64,9 +63,14 @@ private async Task<(string, int)> RunRcsRestApiService()
 
 					while (port > 0)
 					{
-						var directoryName = directory ?? Path.GetDirectoryName(System.Reflection.Assembly.GetExecutingAssembly().Location);
```

Comment:
This is exactly the same code as before, only changed variable name. 

var directoryName = setupDir ?? Path.GetDirectoryName(System.Reflection.Assembly.GetExecutingAssembly().Location);